internal/hmyapi/transactionpool.go

```diff
@@ -55,6 +55,20 @@ func (s *PublicTransactionPoolAPI) GetTransactionsHistory(ctx context.Context, a
 	return ReturnWithPagination(result, args), nil
 }
 
+// GetTransactionsHistoryFull returns the list of transactions with full info that involve a particular address.
+func (s *PublicTransactionPoolAPI) GetTransactionsHistoryFull(ctx context.Context, args TxHistoryArgs) ([]*RPCTransaction, error) {
```

Comment:
do we support pagination ?  This could be too huge of full history.

Reply:
Pagination already applied when tx history hashes are returned.